### DISCUSSION TITLE
请问invalid access code 是什么意思啊，

### DISCUSSION THREAD

**Discussion by flyingcloud2048** [hahahumble/speechgpt#66] [Q&A]
看界面没有地方输入。env 里面也不太对。这个access code 在哪里输入呢
VITE_ACCESS_CODE=REPLACE_WITH_YOUR_OWN
VITE_OPENAI_API_KEY=REPLACE_WITH_YOUR_OWN
VITE_OPENAI_HOST=REPLACE_WITH_YOUR_OWN
VITE_OPENAI_MODEL=REPLACE_WITH_YOUR_OWN
VITE_AWS_REGION=REPLACE_WITH_YOUR_OWN
VITE_AWS_ACCESS_KEY_ID=REPLACE_WITH_YOUR_OWN
VITE_AWS_ACCESS_KEY=REPLACE_WITH_YOUR_OWN
VITE_AZURE_REGION=REPLACE_WITH_YOUR_OWN
VITE_AZURE_KEY=REPLACE_WITH_YOUR_OWN

**Comment by erbanku** [comment]
有地方输入的，如图。如果未找到，请部署最新版。

此外，你需要在部署时提供 `VITE_ACCESS_CODE=REPLACE_WITH_YOUR_OWN`, 才会有如下输入 `访问密码` 框。

![image](https://user-images.githubusercontent.com/79493862/233770367-a0c42a33-8757-4113-91bb-3f7ab6e50754.png)

**Reply by flyingcloud2048** [reply]
谢谢，我前两天时把access code 代码注释后跑的。前端用的少，我直接按yarn /yarn dev ，没有看到access code。是不是readme 少写了如何加这个参数。
对了，现在新版本的azure sdk text to speech 比以前快很多。赞
